platform-hub-api/app/models/ability.rb

```diff
@@ -6,10 +6,18 @@ def initialize(user)
     # IMPORTANT: to understand the caveats when using blocks to specify
     # abilities, see: https://github.com/CanCanCommunity/cancancan/wiki/Defining-Abilities-with-Blocks
 
-    # This will take precedence over everything below!
+
+    # Hub admin role
+    # Note: this will take precedence over everything below!
```

Comment:
Is this comment still relevant, as you can only have one of the roles active now?

Reply:
It's still relevant, but just in the context of how `cancancan` uses this file – i.e. the _ordering_ of these abilities/rules are really important.